### DISCUSSION TITLE
"Date&Time" to "Date & Time" in Settings

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [InfiniTimeOrg/InfiniTime#1837] [Polls]
### Verification

- [X] I searched for similar issues and found none was relevant.

### Introduce the issue

It's a minor formatting change, but it's relatively annoying.

### Version

v1.12.0

**Comment by Elements6007** [comment]
From my personal standpoint "Date&Time" looks better. Adding space makes it look awkward and long. 

![InfiniSim_2023-04-16_210212](https://user-images.githubusercontent.com/88860704/232355356-52b85caa-e019-4eba-8d4a-bf95c0d0bdfc.png)

**Comment by LinuxinaBit** [comment]
> ![InfiniSim_2023-04-16_210212](https://user-images.githubusercontent.com/88860704/232355356-52b85caa-e019-4eba-8d4a-bf95c0d0bdfc.png)

That looks better imo...
I think it only looks a little weird because there aren't any other buttons with 2 words on them on that page.
It matches the first settings page better though.

**Comment by Elements6007** [comment]
> 
> That looks better imo... I think it only looks a little weird because there aren't any other buttons with 2 words on them on that page. It matches the first settings page better though.

True, I guess it does match better.

**Comment by ThatOneCalculator** [comment]
Why not `Date/Time`?
